docs/modeling/connectors/ootb/content.md

```diff
@@ -0,0 +1,367 @@
+| `fields` | *Optional.* Set the list of properties to be returned about the node. | Array | id, name |
+
+### Create a folder
+The `CREATE_FOLDER` action is used to create new folders in the Alfresco Content Services repository.
+
+| Parameter | Description | Type | Example |
+| --------- | ----------- | ---- | ------- |
+| `folderName` | *Required.* A name for the folder. | String | 2020-shipments | 
+| `autorename` | *Optional.* If set to `true` then a suffix will be added to the folder name if a folder with that name already exists.| Boolean | True | 
+| `nodeType` | *Optional.* The node type for the new content. The default value is `cm:file` | String | cm:file | 
+| `include` | *Optional.* Additional information to return about the node. Values include: `properties`, `aspectNames`, `path`, `isLink`, `allowableOperations`, `association`, `isLocked`, `permissions`. | Array | properties, permissions | 
+| `fields` | *Optional.* Set the list of properties to be returned about the node. | Array | id, name |
+
+### Update content
+The `UPDATE_CONTENT` action is used to upload new versions of files to the Alfresco Content Services repository.
+
+| Parameter | Description | Type | Example |
+| --------- | ----------- | ---- | ------- |
+| `name` | *Optional.* A name for the file. | String | January Orders | 
+| `textSource` | *Optional.* The source for the update to the file in text format. | String | | 
+| `fileSource` | *Optional.* The source for the update to the file in [file](../../files.md) variable format. | File | orders-source | 
+| `majorVersion` | *Optional.* If set to `true` then a major version will be created for the file update. If set to `false` a minor version will be created. | Boolean | False | 
+| `comment` | *Optional.* The version comment to appear in the versioning history. | String | Major update to amend order. | 
+| `include` | *Optional.* Additional information to return about the node. Values include: `properties`, `aspectNames`, `path`, `isLink`, `allowableOperations`, `association`, `isLocked`, `permissions`. | Array | properties, permissions | 
+| `fields` | *Optional.* Set the list of properties to be returned about the node. | Array | id, name |
+
+### Move content
+The `MOVE_CONTENT` action is used to move files or folders to a new location in the Alfresco Content Services repository. If a folder is moved then its children will also move to the new location. 
+
+| Parameter | Description | Type | Example |
+| --------- | ----------- | ---- | ------- |
+| `nodeIdTarget` | *Required.* The node ID of the location to move the content to. | String | 164e0082-5ac1-4afa-a09b-619a8c79f66g |
+| `useFiles` | *Optional.* Set to `true` to include files in the search query.  | Boolean | True |
```

Comment:
```suggestion
| `useFiles` | *Optional.* Set to `true` to include only files in the search query.  | Boolean | True |
```

Reply:
Updated all instances as it's in multiple tables - thank you!